docs/versions/latest/Release.md

```diff
@@ -24,6 +24,72 @@
 [//]: # (Optional But higlhy recommended Specify *NONE* if missing)
 [//]: # (#### Relevant Documentation:)
 
+### *Better ordering of the canvas items*
+
+The canvas items are now ordered based on an tree algorithm.
+
+Example:
+
+From input:
+
+```ts
+const nodes = [
+  { name: "A", targets: ["B"] },
+  { name: "B", targets: ["C"] },
+  { name: "C", targets: ["D", "G"] },
+  { name: "D", targets: ["E", "F"] },
+  { name: "E", targets: ["C"] },
+  { name: "F", targets: ["O"] },
+  { name: "G", targets: ["H"] },
+  { name: "H", targets: ["I"] },
+  { name: "I", targets: ["J", "K", "L", "M"] },
+  { name: "J", targets: [] },
+  { name: "K", targets: [] },
+  { name: "L", targets: [] },
+  { name: "M", targets: [] },
+  { name: "O", targets: ["P"] },
+  { name: "P", targets: ["Q"] },
+  { name: "Q", targets: [] },
+
+  // Second start?
+  { name: "S", targets: ["T"] },
+  { name: "T", targets: ["U", "W", "D"] },
+  { name: "U", targets: [] },
+  { name: "W", targets: [] },
+  { name: "X", targets: ["Y"] },
+  { name: "Y", targets: [] },
+];
+```
+
+We get:
+```log
+....................................................................................................
```

Comment:
I know the idea here is to show the node positions, but on the diagram isn't very clear where the start nodes are and which branch a leaf belongs to (e.g. 'J' comes from I but it's on the same line as C)